spec/lib/ruy/rule_set_spec.rb

```diff
@@ -3,25 +3,65 @@
 
 describe Ruy::RuleSet do
 
+  describe 'let' do
+    let(:rule_set) do
+      set = Ruy::RuleSet.new
+
+      set.let :expensive
+      set.let :max
+
+      set.greater_than_or_equal 10, :expensive
+      set.less_than :expensive, 100
```

Comment:
This won't actually try to resolve `:expensive` because it is in the wrong place. It should be `set.less_than 100, :expensive` so that `LessThan` tries to resolve the `let`.